### DISCUSSION TITLE
Curve25519 support?

### DISCUSSION THREAD

**Discussion by itsyigid** [ecies/rs-wasm#20] [Q&A]
Hi!

Is there any chance you'd add Curve25519 support? I would love to open a PR and contribute in my free time. (There is already a Rust crate for Curve25519, https://docs.rs/ecies_25519/latest/ecies_25519/ so I don't think it'd be hard)

**Comment by kigawas** [comment]
I'm planning add it to https://github.com/ecies/rs 
But it's not as easy as js, which already has x25519 and ed25519 support
